builds.yml

```diff
@@ -318,3 +314,10 @@ env:
   # This should NEVER be enabled in production since it slows down react
   ###
   - ENABLE_WHY_DID_YOU_RENDER: false
+
+  ###
+  # Unused environment variables referenced in dependencies
+  # Unset environment variables cause a build error. These are set to `null` to tell our build
+  # system that they are intentionally unset.
+  ###
+  - ETHERSCAN_KEY: null # Used by `gridplus-sdk/dist/util.js`
```

Comment:
It's not being exfiltrated or anything. Gridplus optionally uses it to fetch contract calldata from Etherscan.